lvmd/device_class_manager.go

```diff
@@ -17,6 +17,9 @@ const defaultSpareGB = 10
 //   https://github.com/kubernetes/apimachinery/blob/v0.18.3/pkg/util/validation/validation.go#L42
 var qualifiedNameRegexp = regexp.MustCompile("^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]$")
 
+// This regexp is used to check StripeSize format
+var stripeSizeRegexp = regexp.MustCompile("^([0-9]*)(k|m|g|t|p|K|M|G|T|P)?$")
```

Comment:
I can't find documents describing UNIT.
Would you show me them?

Reply:
fmm, the source code seems https://github.com/lvmteam/lvm2/blob/master/tools/lvmcmdline.c#L565

Reply:
Thanks. I'll fix it. 
Yesterday I looked at https://github.com/lvmteam/lvm2/ but couldn't find it...